### DISCUSSION TITLE
GroupBy.agg

### DISCUSSION THREAD

**Discussion by joshuamosesb** [opensource9ja/danfojs#337] [Q&A]
https://github.com/javascriptdata/danfojs/blob/3688db43b2a5a0b9b4982aa102b47a373a71b29c/danfojs-browser/src/core/groupby.js#L118
along with these `arithemetic` operations, is it feasible to support other aggregation operation of Series, like `nunique`, `min`, `max`?

**Comment by kitsunde** [comment]
I also need a `nunique`, it also seems like pandas allow you to pass an arbitrary function which would be helpful in working around gaps. While `agg` takes a string.
